src/server/exotics/destination/type-of-location/index.js

```diff
@@ -38,8 +40,20 @@ export class TypeOfLocationPage extends QuestionPage {
 
   Answer = Answer
 
-  /** @param {Answer} _answer */
-  nextPage(_answer) {
+  /**
```

Comment:
We haven't added a 'slaughter' option to the RadioButtonConfig, as far as I can tell.  That'll need to be in there!